Fintech_scraper_v01.py

```diff
@@ -1,16 +1,34 @@
+# importing libraries
 import os
-
 import requests
 import pandas as pd
 from bs4 import BeautifulSoup
 
-
-
 # constants (Don't modify)
 base_url = 'https://sifted.eu/european-fintech-startups/'
 excel_filepath = 'fintech.csv'
 excel_filepath_xlsx = 'fintech.xlsx'
 
+# define the common function
+def get_field_by_div(kw,soup):
+    Fintech_table = soup.find_all("div", attrs={"class": "sifted-tile-table__item sifted-tile-table__item"})
+    fintech_list = []
+    for field in Fintech_table:
+        Children = field.findChildren("span", recursive=False)
```

Comment:
```suggestion
Children = field.findChildren("span", recursive=False)
```
It's more typical to use lower case to start variable names